src/popup/router/components/TransferSendForm.vue

```diff
@@ -25,8 +25,9 @@
       new-ui
       :label="$t('modals.send.recipientLabel')"
       :placeholder="$t('modals.send.recipientPlaceholder')"
-      :error-message="addressErrorMsg"
-      :warning-message="addressWarningMsg"
+      :error-message="isTipUrl ? null : addressErrorMsg"
+      :warning-message="isTipUrl ? null : addressWarningMsg"
```

Comment:
I suggest to move this condition that checks the `isTipUrl` to the `addressErrorMsg` and `addressWarningMsg`.